api/app/signals/apps/signals/migrations/0143_rename__user_routingexpression_employee.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.5 on 2021-07-27 12:11
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('signals', '0142_auto_20210727_1406'),
```

Comment:
Please delete these migrations and regenerate so that there is only one migration in this PR. Also please explain the usage of "employee" in stead of "user" - I am not convinced that we should not just use "user" to match other code.